addons/website_forum/tests/test_forum_internals.py

```diff
@@ -145,3 +147,41 @@ def test_tags_usage_unused(self):
 
         self.assertEqual(self.base_forum.tag_most_used_ids, self.base_forum.tag_ids[:MOST_USED_TAGS_COUNT])
         self.assertEqual(self.base_forum.tag_unused_ids, self.env['forum.tag'])
+
+
+@tagged('forum_internals')
+class TestForumController(TestForumCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.forum2, cls.forum3 = cls.env['forum.forum'].create([{'name': f'TestForum{idx + 2}'} for idx in range(2)])
+        cls.post_forum2, cls.post_forum3 = cls.env['forum.post'].create([{
+            'name': 'Introduction',
+            'content': 'This forum is about ...',
+            'forum_id': forum.id,
+        } for forum in (cls.forum2, cls.forum3)])
+        cls.controller = WebsiteForum()
+
+    def _my_other_forums(self, forum):
+        return self.controller._prepare_user_values(forum=forum).get('my_other_forums')
+
+    def test_prepare_user_values_my_other_forum(self):
+        """ Test user other forums values (my_other_forums) in various contexts. """
+        with MockRequest(self.env, website=self.base_website):
+            self.assertFalse(self._my_other_forums(None))
+            self.assertFalse(self._my_other_forums(True))
+            self.assertEqual(self._my_other_forums(self.forum2), self.forum + self.forum3)
+            self.assertEqual(self._my_other_forums(self.forum), self.forum2 + self.forum3)
+            self.assertEqual(self._my_other_forums(self.forum3), self.forum + self.forum2)
+        with self.with_user('employee'), MockRequest(self.env, website=self.base_website):
+            self.assertFalse(self._my_other_forums(self.forum))
+            self.assertFalse(self._my_other_forums(True))
+            self.post_forum2.favourite_ids += self.user_employee
+            self.assertEqual(self._my_other_forums(self.forum), self.forum2)
+            self.user_employee.karma = 3  # required to post
```

Comment:
Can you use the karma rule definition instead of the "magic" number? It would be robust to changes and avoid the need for the extra comment.